packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -14,11 +14,1064 @@ The Okta Administrator Roles API provides operations to manage administrative ro
+  "label": "SF-IT-People",
+  "description": "People in the IT department of San Francisco",
+  "_links": {
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    },
+    "resources": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources"
+    },
+    "bindings": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings"
+    }
+  }
+}
+```
+
+### Get resource set
+<ApiOperation method="get" url="/api/v1/iam/resource-sets/${resourceSetId}" />
+
+Get a resource set by its id
+
+#### Request parameters
+
+| Parameter     | Description                          | Param Type   | DataType                    | Required |
+| :------------ | :----------------------------------- | :----------- | :----------------------------------------------- | :------- |
+| resourceSetId | Unique id of the resource set        | URL          | String                                           | TRUE     |
+
+#### Response parameters
+
+Requested [resource set](#resource-set-object)
+
+#### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+```
+
+#### Response example
+
+```json
+{
+  "id": "iamoJDFKaJxGIr0oamd9g",
+  "label": "SF-IT-People",
+  "description": "People in the IT department of San Francisco",
+  "_links": {
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    },
+    "resources": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources"
+    },
+    "bindings": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings"
+    }
+  }
+}
+```
+
+### List resource sets
+<ApiOperation method="get" url="/api/v1/iam/resource-sets}" />
+
+Get a paginated list of resource sets
+
+#### Response parameters
+
+Paginated list of [resource sets](#resource-set-object)
+
+#### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets"
+```
+
+#### Response example
+
+```json
+{
+  "resource-sets": [
+    {
+      "id": "iamoJDFKaJxGIr0oamd9g",
+      "label": "SF-IT-1",
+      "description": "...",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+        },
+        "resources": {
+          "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources"
+        },
+        "bindings": {
+          "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings"
+        }
+      },
+      ...
```

Comment:
This breaks Github rendering for the rest of this page. Please make this valid JSON.

Reply:
fixed